pkg/controller/config/config.go

```diff
@@ -121,8 +122,9 @@ type WorkqueueConfig struct {
 
 // configuration for a node
 type NodeConfig struct {
-	DefaultDeadlines                DefaultDeadlines `json:"default-deadlines,omitempty" pflag:",Default value for timeouts"`
-	MaxNodeRetriesForSystemFailures int64            `json:"max-node-retries-system-failures" pflag:"2,Maximum number of retries per node for node failure due to infra issues"`
+	DefaultDeadlines               DefaultDeadlines `json:"default-deadlines,omitempty" pflag:",Default value for timeouts"`
+	MaxNodeRetriesOnSystemFailures int64            `json:"max-node-retries-system-failures" pflag:"2,Maximum number of retries per node for node failure due to infra issues"`
+	InterruptibleFailureThreshold  int64            `json:"interruptible-failure-threshold" pflag:"1, number of failures for a node to be still considered interruptible'"`
```

Comment:
lets reword the description. Number of interruptions after which the node is marked as failed?

Reply:
After the threshold is hit, we don't mark the node as failed but don't consider the node as interruptible (eligible for spot). This is to upper bound the number of system failures due to interruption for a particular node.